libpod/networking_linux.go

```diff
@@ -93,6 +94,24 @@ func (r *Runtime) createNetNS(ctr *Container) (err error) {
 	return r.configureNetNS(ctr, ctrNS)
 }
 
+// Configure the network namespace for a rootless container
+func (r *Runtime) setupRootlessNetNS(ctr *Container) (err error) {
+	defer ctr.rootlessSlirpSyncR.Close()
+	defer ctr.rootlessSlirpSyncW.Close()
+
+	path, err := exec.LookPath("slirp4netns")
+	if err != nil {
+		logrus.Errorf("could not find slirp4netns, the network namespace won't be configured")
```

Comment:
You should return the error, Could you get an error like Permission denied or some other error?

Reply:
yes, it seems that it can return an error like that.  Pushed a new version that doesn't ignore `err`